### PR TITLE
Added examples for creating distributed clusters in BigAnimal for AWS and GCP

### PR DESCRIPTION
**Pull request by dwicinas**
Two new examples

Addresses https://enterprisedb.atlassian.net/browse/DOCS-165

### CODE REVIEW

**Review by nidhibhammar — APPROVED** [review]
LGTM

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 1]
```diff
@@ -457,6 +458,138 @@ witnessGroups:
       maintenanceStartDay: monday
 ```
 
+#### AWS example
+
+```
+clusterName: biganimal-aws-cluster                                      # <string:  cluster name>
+password: password@12345                                              # <string:  cluster password (must be at least 12 characters)>
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:465]: lets keep it same across all examples i.e copy from `Azure example`

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 2]
```diff
@@ -457,6 +458,138 @@ witnessGroups:
...
+    customMaintenanceWindow:                                          # <Object:  Schedule a custom maintenance window for witness group>
+      maintenanceStartTime: 18:00                                     # <string:  Set a start time in 24 hour format in UTC (e.g. 15:00) of day to initiate>
+      maintenanceStartDay: monday                                     # <string:  Select desired day (e.g. monday) from a week for maintenance activities>
+```
+
+#### Google Cloud example
+
+```
+clusterName: biganimal-gcp-cluster                                    # <string:  cluster name>
+password: password@12334                                                # <string:  cluster password (must be at least 12 characters)>
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:531]: lets keep it same across all examples i.e copy from `Azure example`

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 3]
```diff
@@ -403,7 +402,9 @@ The syntax of the command is:
 biganimal pgd create --config-file <config_file>
 ```
 
-Where `<config_file>` is a valid path to a YAML configuration file. For example:
+Where `<config_file>` is a valid path to a YAML configuration file. 
+
+#### Azure example
 
 ```
 clusterName: pgd-cli-name
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:410]: biganimal-azure-pgd-cluster

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 4]
```diff
@@ -457,6 +458,138 @@ witnessGroups:
       maintenanceStartDay: monday
 ```
 
+#### AWS example
+
+```
+clusterName: biganimal-aws-cluster                                      # <string:  cluster name>
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:464]: biganimal-aws-pgd-cluster

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 5]
```diff
@@ -457,6 +458,138 @@ witnessGroups:
...
+    region: australiaeast                                                   # <string:  provider region id>
+    customMaintenanceWindow:                                          # <Object:  Schedule a custom maintenance window for witness group>
+      maintenanceStartTime: 18:00                                     # <string:  Set a start time in 24 hour format in UTC (e.g. 15:00) of day to initiate>
+      maintenanceStartDay: monday                                     # <string:  Select desired day (e.g. monday) from a week for maintenance activities>
+```
+
+#### Google Cloud example
+
+```
+clusterName: biganimal-gcp-cluster                                    # <string:  cluster name>
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:530]: biganimal-gcp-pgd-cluster

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 1 above)
Comment (dwicinas) [reply product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx]: Applied suggested edit

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 4 above)
Comment (dwicinas) [reply product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx]: Applied suggested edit

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 5 above)
Comment (dwicinas) [reply product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx]: Applied suggested edit

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 2 above)
Comment (dwicinas) [reply product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx]: Applied suggested edit

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Pull request merged**